### ISSUE TITLE
Glide.js

### ISSUE THREAD

**Issue by vorvek** [DylanHalstead/PolyChord-Website#10]
Am I correct assuming this is only used in the home page? Currently the JS is loaded in the footer of every page. It's not too important, as it doesn't break anything, nor does it load a second time as visiting the page once caches the script, but there's an ugly error in the console.

**Comment by DylanHalstead** [comment]
Yeah just the home page, it was originally used for a carousel but they asked for it to be removed. I still think it's good to have the option so kept the setup for the code so it could be turned into a carousel if wanted.
